foxi/onnxifi.h

```diff
@@ -864,21 +864,28 @@ typedef struct onnxTensorDescriptorV1 {
    */
   const uint64_t* shape;
   /**
-   * A flag indicates whether this tensor is a quantized tensor or not.
-   * If it is a quantized tensor, the value should be set to 1.
-   * By default it should be set to 0.
+   * This indicates how many quantization parameters this tensor has in the
+   * scales and biases fields. If this tensor is not quantized then this is 0.
+   * This must be less than or equal to the size of the tensor in the 
+   * quantizationAxis dimension. Note that quantization paramaters will be
+   * shared amongst equally sized groups along the quantizationAxis.
    */
-  uint8_t is_quantized;
+  uint64_t quantizationParams;
+  /**
+   * Indicates the axis along which quantization parameters are shared.
+   */
+  uint32_t quantizationAxis;
```

Comment:
nit: put `quantizationAxis` before `quantizationParams` so that we know that `quantizationParams` specifies the lengths of scales and biases which follows immediately after?

Reply:
Sounds good